daemon/cluster/cluster.go

```diff
@@ -354,9 +351,6 @@ func (c *Cluster) Join(req types.JoinRequest) error {
 	for {
 		select {
 		case <-certificateRequested:
```

Comment:
Should be safe to remove this whole case, certificateRequested var and for cycle.